src/components/MaterialUI/DataDisplay/PredefinedElements/Placeholder.js

```diff
@@ -20,11 +21,55 @@ const useStyles = makeStyles(theme => ({
 		fontSize: theme.typography.h3Size,
 		color: theme.palette.grey.icon,
 	},
+	placeholderRow: {
+		alignSelf: "flex-start",
+		width: "100%",
+		display: "flex",
+		borderBottom: `1px solid ${theme.palette.grey.borders}`,
+		backgroundColor: `${theme.palette.background.paper}!important`,
+	},
+	placeholderCell: {
+		display: "inline-flex",
+		padding: theme.spacing(2.7, 1.6),
+		paddingRight: theme.spacing(4.6),
+		flex: 1,
+		alignItems: "center",
+	},
+	textSkeleton: {
+		width: "100%",
+		height: 6,
+		animation: false,
+	},
+	chipSkeleton: {
+		width: 60,
+		height: 25,
+		animation: false,
+		borderRadius: "15px",
+	},
 }));
 
-const Placeholder = ({ icon, title, subtitle }) => {
+const Placeholder = ({ icon, title, subtitle, cellList = [] }) => {
 	const classes = useStyles();
-	return (
+	return cellList.length ? (
+		<div className={classes.placeholderRow}>
+			{cellList.map((value, index) => {
+				switch (value) {
+					case "status":
```

Comment:
Let's name it not status but "chip" as generaly we use here MUI Chip component. Thanks